src/common/types/IBufferUtils.ts

```diff
@@ -12,7 +12,7 @@ export default interface IBufferUtils<Bufferlike, Output, ToBufferOutput, WordAr
   hexDecode: (string: string) => Output;
   utf8Encode: (string: string) => Output;
   utf8Decode: (buffer: Bufferlike) => string;
-  bufferCompare: (buffer1: Bufferlike, buffer2: Bufferlike) => number;
+  areBuffersEqual: (buffer1: Bufferlike, buffer2: Bufferlike) => boolean;
```

Comment:
don't really mind this change but worth mentioning - compare functions are a thing in javascript because they're used as [a parameter for Array.sort](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort#parameters)

Reply:
Ah, I see — I had suspected it might be something like that, but given we're not using the compare functionality anywhere I reckon the change is OK?

Reply:
yep, that's fine by me 👍